xarray/backends/api.py

```diff
@@ -488,6 +488,10 @@ def open_dataset(
           as coordinate variables.
         - "all": Set variables referred to in  ``'grid_mapping'``, ``'bounds'`` and
           other attributes as coordinate variables.
+
+        Only existing variables can be set as coordinates. Missing variables
```

Comment:
I hope that this is short enough but explains how missing variables are treated. No native speaker here, so please suggest better wording/grammar.